src/java/main/org/apache/zookeeper/ZooKeeper.java

```diff
@@ -2347,25 +2347,25 @@ public void getACL(final String path, Stat stat, ACLCallback cb,
 
     /**
      * Set the ACL for the node of the given path if such a node exists and the
-     * given version matches the version of the node. Return the stat of the
+     * given aclVersion matches the aclVersion of the node. Return the stat of the
      * node.
      * <p>
      * A KeeperException with error code KeeperException.NoNode will be thrown
      * if no node with the given path exists.
      * <p>
      * A KeeperException with error code KeeperException.BadVersion will be
-     * thrown if the given version does not match the node's version.
+     * thrown if the given aclVersion does not match the node's aclVersion.
      *
      * @param path
      * @param acl
-     * @param version
+     * @param aclVersion
```

Comment:
ok..done.. But there are so many (example : org.apache.zookeeper.ZooKeeper#getChildren(java.lang.String, boolean, Stat)) not updated the params..